### PR TITLE
Refactor Helm Charts as per the reconfiguration/parametrization of paths in source code and Docker images

### PR DESCRIPTION
**Pull request by gab-arrobo**
@thakurajayL,
Should we change the name (extension) of the config files to `yaml` instead of `conf` because we already know that they are yaml files?

### CODE REVIEW

**Comment by gab-arrobo** [comment]
> OK to change it to yaml

In this same PR? Or in another/new PR?

**Comment by thakurajayL** [comment]
If possible same PR would be good to avoid multiple review

**Comment by gab-arrobo** [comment]
> If possible same PR would be good to avoid multiple review

Done. PR now includes change of extension of configuration files from `conf` to `yaml`

File: 5g-control-plane/templates/bin/_upf-adapter-run.sh.tpl [hunk 1]
```diff
@@ -7,11 +7,14 @@
 set -xe
 
 {{- if .Values.config.coreDump.enabled }}
-cp /aether/upfadapter /tmp/coredump/
+cp /usr/local/bin/upfadapter /tmp/coredump/
 {{- end }}
 
-cd /aether
+CFGPATH=/home
+FILENAME=config.yaml
```
Comment (thakurajayL) [inline 5g-control-plane/templates/bin/_upf-adapter-run.sh.tpl:14]: all config files name has NF in it. e.g. amf config is amfcfg.yaml .. Can we not keep same for upf adapter?

File: 5g-ran-sim/templates/bin/_gnbsim-run.sh.tpl [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
 
 {{- if .Values.config.coreDump.enabled }}
-cp /gnbsim/bin/gnbsim /tmp/coredump/
+cp /usr/local/bin/gnbsim /tmp/coredump/
 {{- end }}
 
-cd /gnbsim
-cat ./config/gnb.conf
+CFGPATH=/home
+FILENAME=gnb.conf
```
Comment (thakurajayL) [inline 5g-ran-sim/templates/bin/_gnbsim-run.sh.tpl:15]: why not yaml here?

File: 5g-control-plane/templates/bin/_upf-adapter-run.sh.tpl (hunk 1 above)
Comment (gab-arrobo) [reply 5g-control-plane/templates/bin/_upf-adapter-run.sh.tpl]: Done.

File: 5g-ran-sim/templates/bin/_gnbsim-run.sh.tpl (hunk 2 above)
Comment (gab-arrobo) [reply 5g-ran-sim/templates/bin/_gnbsim-run.sh.tpl]: Done

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**